core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -1149,6 +1149,19 @@
 
   public static final String USE_KETTLE_DEFAULT = "false";
 
+  /**
+   * Sorts the data in batches and writes the batch data to store with index file.
+   */
+  public static final String LOAD_USE_BATCH_SORT = "carbon.load.use.batch.sort";
+
+  public static final String LOAD_USE_BATCH_SORT_DEFAULT = "true";
```

Comment:
Better to make it `false` as of now, and revisit it after checking the performance and loading and reading. 
And please add comment mentioning something like "if fast loading is favored, set it to true. If fast query performance is favored, set it to false"